modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm

```diff
@@ -248,3 +248,22 @@
 	new /obj/item/grenade/spawnergrenade/manhacks(src)
 	for(var/i in 1 to 6)
 		new /obj/item/pen/edagger(src)
+
+/obj/item/storage/box/syndie_kit/loadout/greytide/PopulateContents() //hope you like reading baby, this kit's all about little initial power for massive endgame potential- you have staying power with energy weapons and other armaments but need to find or make them yourself, so GET TIDING
+	new /obj/item/spear/grey_tide(src)
+	new /obj/item/book/granter/crafting_recipe/pipegun_prime(src)
+	new /obj/item/book/granter/crafting_recipe/trash_cannon(src)
+	new /obj/item/construction/rcd/loaded/upgraded(src)
+	for(var/i in 1 to 3)
+		new /obj/item/rcd_ammo/large(src)
+	new /obj/item/clothing/under/color/grey/ancient(src)
+	new /obj/item/clothing/mask/gas(src)
+	for(var/i in 1 to 3)
+		new /obj/item/gas_filter(src)
+	new /obj/item/storage/belt/chameleon(src)
+	new /obj/item/storage/toolbox/mechanical/old/clean(src) //contains insuls for the aspiring god of tiding
```

Comment:
This is the toolbox that costs 2 TC and deals more damage the more TC it has in it. This doesn't make much sense if there's not really much TC left for the traitor after the fact, and it feels conflicting with the spear.

Reply:
good idea

Reply:
however, the whole point is that the toolbox is the only one that comes with a full toolset plus insulated gloves, i understand the tc gimmick but it need not apply here

Reply:
You can make a new toolbox like that, that doesn't have the whole TC functionality.

Reply:
Why make a whole new subtype just for that? The cost doesn't leave room anyways